lib/src/widgets/custom_image_crop_widget.dart

```diff
@@ -195,21 +209,30 @@ class _CustomImageCropState extends State<CustomImageCrop>
   }
 
   void onScaleUpdate(ScaleEvent event) {
+    if (widget.disableGestureScale) return;
```

Comment:
This breaks the canRotate, if canRotate is true, but disableGestureScale is also true, the rotation is also disabled or is this the intention?